spring-cloud-gcp-autoconfigure/pom.xml

```diff
@@ -22,20 +22,112 @@
             <artifactId>spring-cloud-gcp-core</artifactId>
             <optional>true</optional>
         </dependency>
-        <!-- Configuration processor -->
+
+        <!-- Global -->
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-configuration-processor</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>commons-logging</groupId>
+            <artifactId>commons-logging</artifactId>
+        </dependency>
+
         <dependency>
             <groupId>org.springframework.cloud</groupId>
             <artifactId>spring-cloud-gcp-config</artifactId>
             <optional>true</optional>
         </dependency>
+
+        <!-- Logging -->
         <dependency>
-            <groupId>commons-logging</groupId>
-            <artifactId>commons-logging</artifactId>
+            <groupId>org.springframework.cloud</groupId>
+            <artifactId>spring-cloud-gcp-logging</artifactId>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>javax.servlet-api</artifactId>
+            <scope>test</scope>
+        </dependency>
+
+        <!-- Pub/Sub -->
+        <dependency>
+            <groupId>org.springframework.cloud</groupId>
+            <artifactId>spring-cloud-gcp-pubsub</artifactId>
+            <optional>true</optional>
+        </dependency>
+
+        <!-- Cloud SQL -->
+
+        <!-- HikariCP used as default DataSource -->
+        <dependency>
+            <groupId>com.zaxxer</groupId>
+            <artifactId>HikariCP</artifactId>
+            <scope>compile</scope>
```

Comment:
Should it be `optional` like everything else?

Reply:
Good catch.